src/GDGUkraine/templates/card.html

```diff
@@ -41,7 +91,9 @@
             <h3>Registration to "{{ event.title }}"</h3>
             <div>{{ event.desc }}</div>
         </td>
-            <td class="img" rowspan="{{ registration.fields | length + 14 + 1 }}"><img src="https://chart.googleapis.com/chart?chs=300x300&cht=qr&chl={{ qrdata }}&choe=UTF-8" /></td>
+            <td class="img" rowspan="{{ registration.fields | length + 14 + 1 }}"><img src="https://chart.googleapis.com/chart?chs=300x300&cht=qr&chl={{ qrdata }}&choe=UTF-8" />
+                {% if is_admin %} {% if registration.visited %} <button disabled="disabled" class="btn btn-success">Visited</button> {% else %}<button class="btn approve-btn" onclick="approve_visit()">Approve visit</button> <span class="result"></span>{% endif %}{% endif %}
```

Comment:
And why do you display button in case visit has already been approved?
Why not just a corresponding message?